pkg/cmd/roachtest/backup.go

```diff
@@ -82,120 +79,6 @@ func registerBackup(r *testRegistry) {
 		},
 	})
 
-	KMSSpec := makeClusterSpec(3)
-	r.Add(testSpec{
-		Name:       fmt.Sprintf("backup/KMS/%s", KMSSpec.String()),
-		Owner:      OwnerBulkIO,
-		Cluster:    KMSSpec,
-		MinVersion: "v20.2.0",
```

Comment:
I assume you've done this already but we basically want to remove _all_ files that had a minimum version less as 20.2 (and above).

Reply:
ack, I had missed a few going to list all the files and roachtests I have deleted in the commit message.